### ISSUE TITLE
PVector.normalize() does not return PVector

### ISSUE THREAD

**Issue by bennyty** [processing-js/processing-js#207]
This means that you cannot chain functions together.

For example:
steerForce.normalize().mult(maxSpeed).sub(velocity).limit(maxHandling);

Throws "Uncaught TypeError: Cannot read property 'mult' of undefined"

**Comment by Pomax** [comment]
yeah, there's a PR open for that (#201) but it hasn't been updated in a way that can land yet
